ts/src/binance.ts

```diff
@@ -3135,6 +3183,15 @@ export default class binance extends Exchange {
             }
         }
         const isMarginTradingAllowed = this.safeBool (market, 'isMarginTradingAllowed', false);
+        let marginMode = undefined;
+        if (spot) {
```

Comment:
if `swap` can't we assume `isolated/cross` margin as `true` ?

Reply:
hum. initially i was about spot-margin markets, not  about swaps (because binance perps have both cross & isolated probably for all symbols). however, good note, i think we can have them for contracts too